backend/mora/service/detail_writing.py

```diff
@@ -393,6 +393,47 @@ def edit():
     Request payload contains a list of edit objects, each differentiated
     by the attribute ``type``. Each of these object types are detailed below:
 
+    **Employee**:
+
+    :<json string type: **"employee"**
+    :<json string uuid: The UUID of the employee,
+    :<json object original: An **optional** object containing the original
+        state of the employee to be overwritten. If supplied, the change
+        will modify the existing registration on the employee object.
+        Detailed below.
+    :<json object data: An object containing the changes to be made to the
+        employee. Detailed below.
+
+    The **original** and **data** objects follow the same structure.
+    Every field in **original** is required, whereas **data** only needs
+    to contain the fields that need to change along with the validity dates.
+
+    :<jsonarr string cpr_no: The cpr number of the employee
+    :<jsonarr string name: The Name of the employee
+    :<jsonarr object validity: The validities of the changes.
+
+    .. sourcecode:: json
+        [{
```

Comment:
In order for Sphinx to pick up the directory, you need a newline between the directive and the text. Then, you'll get a warning about your JSON not being actually valid JSON. (`None` ≠ `null`; no final commas; missing quotes.)

To build the docs, use `./flask.sh docs`.